src/woo_publications/publications/api/filters.py

```diff
@@ -45,7 +49,28 @@ class PublicationFilterSet(FilterSet):
             "verkorte_titel",
         ),
     )
+    eigenaar = filters.CharFilter(
+        help_text=_("Filter publications based on the owner identifier of the object."),
+        method="filter_eigenaar",
+    )
 
     class Meta:
         model = Publication
-        fields = ("sorteer",)
+        fields = (
+            "sorteer",
+            "eigenaar",
+        )
+
+    def filter_eigenaar(self, queryset, name: str, value: str):
+        publication_ct = ContentType.objects.get(
+            app_label="publications",
+            model="publication",
+        )
+
+        publication_object_ids = TimelineLogProxy.objects.filter(
+            content_type=publication_ct,
+            extra_data__event=Events.create,
+            extra_data__acting_user__identifier=value,
+        ).values_list("object_id", flat=True)
```

Comment:
I'll allow this for now, but I am a little bit concerned about the model being used directly rather than through helper functions. The `extra_data` internal structure is really something that should be contained within `woo_publications.logging`. and not leak outside. E.g. a nicer Python API would be:

```py
from woo_publications.logging.service import filter_logs 

logs = filter_logs(content_type=publication_ct, event=Events.create, acting_user=value)
publication_object_ids = logs.values_list("object_id", flat=True)
```

no need to make this change now, let's first see if this approach survives first contact :sweat_smile: